modules/cluster-logging-collector-log-forward-es.adoc

```diff
@@ -65,7 +65,7 @@ spec:
 <3> Specify a name for the output.
 <4> Specify the `elasticsearch` type.
 <5> Specify the URL and port of the external Elasticsearch instance as a valid absolute URL. You can use the `http` (insecure) or `https` (secure HTTP) protocol. If the cluster-wide proxy using the CIDR annotation is enabled, the output must be a server name or FQDN, not an IP Address.
-<6> If using an `https` prefix, you must specify the name of the secret required by the endpoint for TLS communication. The secret must exist in the `openshift-logging` project and must have keys of: *tls.crt*, *tls.key*, and *ca-bundle.crt* that point to the respective certificates that they represent.
+<6> If using an `https` prefix and the external Elasticsearch server is using a private CA or you enabled mutual Transport Layer Security (mTLS) to Fluentd, you must specify the name of the secret required by the endpoint for TLS communication. To trust the Elasticsearch private CA, you must specify the key *ca-bundle.crt*. To enable mTLS, you must specify keys: *ca-bundle.crt*, *tls.crt*, and *tls.key*. The secret must exist in the `openshift-logging` project and must be of type `opaque` or `generic`.
```

Comment:
```suggestion
<6> If you are using HTTPS and the external Elasticsearch server is using a private CA or mutual Transport Layer Security (mTLS), specify the name of the secret required by the endpoint. For the Elasticsearch private CA, specify the name of the *ca-bundle.crt* private key. For mTLS, specify the name of the *ca-bundle.crt*, *tls.crt*, or *tls.key* key. The secret must exist in the `openshift-logging` project and must be of type `opaque` or `generic`.
```

Reply:
@vparfonov @igor-karpukhin Does this proposed revision make sense?

Reply:
@rolfedh looks good